cynic/src/integrations/chrono.rs

```diff
@@ -25,6 +25,26 @@ impl Scalar for NaiveDate {
 
 crate::impl_serializable_argument_for_scalar!(NaiveDate);
 
+impl Scalar for NaiveTime {
+    fn decode(value: &serde_json::Value) -> Result<Self, DecodeError> {
+        match value {
+            serde_json::Value::String(s) => {
+                Ok(NaiveTime::parse_from_str(s, "%H:%M").map_err(chrono_decode_error)?)
```

Comment:
Could you try changing this to just `s.parse().map_err(...)` please?  _hoping_ that it does the right thing when seconds are missing but also supports seconds.